packages/augur-core/source/contracts/uniswap/interfaces/IUniswapV2.sol

```diff
@@ -1,6 +1,26 @@
-pragma solidity 0.5.10;
+pragma solidity 0.5.15;
 
 
 interface IUniswapV2 {
```

Comment:
Since Uniswap consists of two contracts, consider renaming this interface to `IUniswapExchange` for clarity.